Assets/HoloToolkit/Input/Scripts/ControllerInfo.cs

```diff
@@ -219,5 +217,119 @@ public void LoadInfo(Transform[] childTransforms, ControllerVisualizer visualize
                 }
             }
         }
+
+        public void AnimateGrasp(bool isGrasped)
+        {
+            if (grasp != null && graspPressed != null && graspUnpressed != null && isGrasped != wasGrasped)
+            {
+                if (isGrasped)
+                {
+                    grasp.transform.localPosition = graspPressed.localPosition;
```

Comment:
I see this pattern a lot in the code. You could consider doing something like:

private void SetButtonPositonRotation(GameObject controllerButton, Transform controllerButtonTransform)
{
controllerButton.transform.localPosition = controllerButtonTransform.localPosition;
rotation blah
}

You could make this even better by passing in structs/class for each button type as most are similar calls.